content/en/docs/tasks/traffic-management/ingress/secure-ingress/index.md

```diff
@@ -510,21 +420,11 @@ $ kubectl create -n istio-system secret generic httpbin-credential --from-file=k
 
     {{< /text >}}
 
-1. Instead of creating a `httpbin-credential` secret to hold all the credentials, you can
-   create two separate secrets:
+Istio supports reading a few different formats, to support integration with various tools such as [cert-manager](/docs/ops/integrations/certmanager/):
```

Comment:
Istio supports several formats of what? I assume a config file or secrets file?

Reply:
Yeah this would be the Secret object. So Ill reword this to `Istio supports reading a few different formats of Secrets ....`